PYME/cluster/HTTPDataServer.py

```diff
@@ -897,7 +897,13 @@ def main(protocol="HTTP/1.0"):
     os.chdir(options.root)
 
     if options.advertisements == 'local':
-        ns = sqlite_ns.getNS('_pyme-http')
+        # preference is to avoid zeroconf on clusterofone due to poor
+        # performance on windows on crowded networks (for oldish versions at least)
```

Comment:
```suggestion
        # performance on windows on crowded networks
```
performance might be improved in more recent versions, but it will still be an issue - it's fundamentally not the zeroconf modules fault but rather a number of poorly behaved clients that generate excessive traffic. The only real solution is to run zeroconf on another port.

Reply:
also not strictly windows specific (although potentially worse on windows)

Reply:
Worst case I've encountered so far is a network with a) lots of windows machines, b) lots of itunes installs on windows, and c) lots of labview installs, as each of these generates a bunch of excess mDNS traffic.